packages/zowe-explorer/__tests__/__unit__/shared/utils.unit.test.ts

```diff
@@ -1085,41 +1101,200 @@ describe("Shared utils unit tests - function promptForEncoding", () => {
     });
 });
 
-describe("Shared utils unit tests - function confirmForUnsavedDoc", () => {
```

Comment:
I believe this `describe` block should be kept - we want to keep both sets of tests for `confirmForUnsavedDoc` and `initializeFileOpening`.